__tests__/index.js

```diff
@@ -384,48 +444,6 @@ testRule(rule, {
       column: 19,
       description: "float: right rules out baseline"
     },
-    {
-      code: "a { display: table-cell; margin: 10px; }",
```

Comment:
This tests are moved to below where I generate `table-*` tests for `margin` and `padding`.